src/main/java/school/faang/sprint1/task_44656/HogwartsSpells.java

```diff
@@ -0,0 +1,49 @@
+package school.faang.sprint1.task_44656;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HogwartsSpells {
+
+    public Map<Integer, SpellEvent> spellById = new HashMap<>();
+    public Map<String, List<SpellEvent>> spellsByType = new HashMap<>();
+
+    public void addSpellEvent(int id, String eventType, String actionDescription) {
+        SpellEvent spellEvent = new SpellEvent(id, eventType, actionDescription);
+        spellById.put(spellEvent.getId(), spellEvent);
+        spellsByType.put(spellEvent.getEventType(), new ArrayList<>());
```

Comment:
можно написать так 
        spellsByType.put(spellEvent.getEventType(), new ArrayList<>()).add(spellEvent);

Reply:
выдаёт ошибку

Reply:
пришли ошибку плиз

Reply:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.List.add(Object)" because the return value of "java.util.Map.put(Object, Object)" is null
at school.faang.sprint1.task_44656.HogwartsSpells.addSpellEvent(HogwartsSpells.java:17)
at school.faang.sprint1.task_44656.Main.main(Main.java:12)